code/game/objects/objs.dm

```diff
@@ -363,6 +363,9 @@ a {
 /obj/proc/cult_reveal() //Called by cult reveal spell and chaplain's bible
 	return
 
+/obj/proc/is_cleanable() //Called when you want to clean something, and usualy delete it after
```

Comment:
Just pointing out that the cleaning methods filter by `/obj/effect` so anything not of that subtype won't even be considered for cleaning.

Reply:
Scope this to the `/obj/effect` level please.